specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -7521,10 +7521,35 @@
         "triggerThreshold": {
           "description": "The threshold triggers this alert rule.",
           "type": "integer"
+        },
+        "eventGroupingSettings": {
```

Comment:
add new property in output is breaking change, which requires a new version https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-changes-that-require-a-version-change.  Here's process of breaking hcange policy https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/43942/Policy-for-Breaking-Changes

Reply:
this property is optional - this is not breaking change and this property is implemented in production already for a long time without any impact

Reply:
yes, even optional property in output will be breaking change in stable version. thanks for share that this property is already in production. That means this PR is a swagger bug fixing.